source/tutorials/sample-classifier.rst

```diff
@@ -83,6 +83,13 @@ In addition to the predicted class information, the model also reports the indiv
      --m-input-file moving-pictures-classifier/probabilities.qza \
      --o-visualization moving-pictures-classifier/probabilities.qzv
 
+Besides outputting the predictions and probabilities of the test samples, the pipeline also returns the true target values of all train and test samples in ``training_targets.qza`` and ``test_targets.qza`` respectively. Both are ``SampleData[TrueTargets]`` artifacts that can be viewed as metadata and inspected with ``metadata tabulate``, e.g. for ``test_targets.qza``:
+
+.. command-block::
+
+   qiime metadata tabulate \
+     --m-input-file moving-pictures-classifier/test_targets.qza \
```

Comment:
you could view the predictions and true targets side-by-side for an even more interesting output! Just add the predictions artifact as another `--m-input-file`